docs/DESTROY.md

```diff
@@ -3,7 +3,7 @@
 You can shutdown/destroy all resources from your environment by running this command below:
```

Comment:
Use the following command to shutdown/destroy all resources from your environment.

Reply:
Done